dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerAPI.java

```diff
@@ -48,6 +49,9 @@
 	 */
 	public Container copy(Container source, Host destination, User user, boolean respectFrontendRoles) throws DotDataException, DotSecurityException;
 
+	public Optional<Container> findContainer(String idOrPath, User user, boolean live, boolean respectFrontendRoles)

```

Comment:
Codacy found an issue: [Unnecessary modifier 'public' on method 'findContainer': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)